src/FixerConfiguration/SubsetOf.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\FixerConfiguration;
+
+/**
+ * @internal
+ */
+final class SubsetOf
```

Comment:
of what?


when I will have later, `function foo(SubsetOf $subset) {}` I have no idea what is this subset of..

Reply:
Of whatever you give it when instantiating it, e.g. `new SubsetOf(['foo', 'bar'])`.

Reply:
but you know that only during instantiating, not when you use it.

Reply:
like here, `is value subset of.... what?`
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3669/files#diff-88cad9ab17d8e3b415e648e3fd31f3d2R176

Reply:
Right... `AllowedValueSubset`?

Reply:
`SubsetOf` renamed to `AllowedValueSubset`.